### PR TITLE
feature(ui-ux): loans v2 - header and carousel

### PR DESCRIPTION
**Pull request by chloezxyy**
#### What this PR does / why we need it:
- Update the header of the Loans page so it conforms to Loans 2.0 design
- Update empty state for Vaults

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

**Review by pierregee — CHANGES_REQUESTED** [review]

File: mobile-app/app/hooks/useNavigatorHeaderStylesOption.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { StackNavigationOptions } from "@react-navigation/stack";
```
[the remark this answers is not in the document]
Comment (chloezxyy) [reply mobile-app/app/hooks/useNavigatorHeaderStylesOption.tsx:1]: Yep we can do that, we'll shift the hook into the location that you've mentioned, but for the reusing code we can do it on another PR instead?

File: mobile-app/app/screens/WalletNavigator/screens/components/PaginationButton.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      }}
+      dark={tailwind("border-mono-dark-v2-900")}
+      light={tailwind("border-mono-light-v2-900")}
+      style={tailwind("rounded-2xl-v2 text-center py-2 px-4 border", {
+        "bg-black": endOfPagination && isLight,
+        "bg-white": endOfPagination && !isLight,
+      })}
+    >
+      <ThemedTextV2
+        style={tailwind("font-normal-v2 text-center text-base")}
```
Comment (JoshuaC817) [inline mobile-app/app/screens/WalletNavigator/screens/components/PaginationButton.tsx:67]:
```suggestion
        style={tailwind("font-semibold-v2 text-center text-base")}
```

File: mobile-app/app/screens/WalletNavigator/screens/components/PaginationButton.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        if (curIndex < PAGINATION_END + 1) {
+          return goToNextPage(curIndex + 1);
+        }
+        if (endOfPagination) {
+          return dismissModal(); // @chloe TODO: should close bottom sheet?
+        }
+      }}
+      dark={tailwind("border-mono-dark-v2-900")}
+      light={tailwind("border-mono-light-v2-900")}
+      style={tailwind("rounded-2xl-v2 text-center py-2 px-4 border", {
```
Comment (JoshuaC817) [inline mobile-app/app/screens/WalletNavigator/screens/components/PaginationButton.tsx:61]:
```suggestion
      style={tailwind("rounded-2xl-v2 text-center py-3.5 px-4 border", {
```

File: mobile-app/app/hooks/useNavigatorHeaderStylesOption.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (pierregee) [reply mobile-app/app/hooks/useNavigatorHeaderStylesOption.tsx]: can do at same PR

File: mobile-app/app/screens/WalletNavigator/screens/components/PaginationButton.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        if (curIndex < PAGINATION_END + 1) {
+          return goToNextPage(curIndex + 1);
+        }
+        if (endOfPagination) {
+          return dismissModal(); // @chloe TODO: should close bottom sheet?
+        }
+      }}
+      dark={tailwind("border-mono-dark-v2-900")}
+      light={tailwind("border-mono-light-v2-900")}
+      style={tailwind("rounded-2xl-v2 text-center py-3.5 px-4 border", {
```
Comment (JoshuaC817) [inline mobile-app/app/screens/WalletNavigator/screens/components/PaginationButton.tsx:61]:
```suggestion
      style={tailwind("rounded-full text-center py-3.5 px-4 border", {
```

**Review by pierregee — APPROVED** [review]

**Pull request merged**